zenguard_library.ipynb

```diff
@@ -0,0 +1,184 @@
+﻿{
+  "nbformat": 4,
+  "nbformat_minor": 0,
+  "metadata": {
+    "colab": {
+      "provenance": [],
+      "toc_visible": true
+    },
+    "kernelspec": {
+      "name": "python3",
+      "display_name": "Python 3"
+    },
+    "language_info": {
+      "name": "python"
+    }
+  },
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "view-in-github",
+        "colab_type": "text"
+      },
+      "source": [
+        "<a href=\"https://colab.research.google.com/github/ZenGuard-AI/fast-llm-security-guardrails/blob/main/zenguard_library.ipynb\" target=\"_parent\"><img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/></a>"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "## Zenguard: Prompt Injection Detection with Library"
+      ],
+      "metadata": {
+        "id": "ouO2X6oz9uNR"
+      }
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "### Install the **zenguard** package:"
+      ],
+      "metadata": {
+        "id": "OiMYRN6X-VzA"
+      }
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "!pip install zenguard"
+      ],
+      "metadata": {
+        "id": "rt-akQwe7u5j",
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        },
+        "outputId": "b9626612-e6dc-4bf7-cfa9-f686d5db2973"
+      },
+      "execution_count": null,
+      "outputs": [
+        {
+          "output_type": "stream",
+          "name": "stdout",
+          "text": [
```

Comment:
do we really need to have outputs here? especially the ones that just download stuff?